src/functions/types.ts

```diff
@@ -187,13 +188,13 @@ export type FunctionDefinitionArgs<
   /** An optional description for your function. */
   description?: string;
   /** An optional map of input parameter names containing information about their type, title, description, required and (additional) properties. */
-  "input_parameters"?: {
-    properties: InputParameters;
-    required: RequiredInput;
-  };
+  "input_parameters"?: ParameterPropertiesDefinition<
```

Comment:
I found a place to re-use this existing Type we had, so wanted to make things a bit more consistent. I did test that this still works for function-related code.